components/button/BookmarkButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Button } from 'components/button'; 
+
+interface Props {
```

Comment:
let's remove the children since these buttons will never use children, this way we can remove the interface entirely